DataProcessing/GetGOTerms.m

```diff
@@ -11,12 +11,14 @@
 end
 %-------------------------------------------------------------------------------
 
-dbc = ConnectMeDatabase();
+% Replace following with path to the db of your GO release created in sqlite3
+% (following the format of GODaily_2021-01-25.sql used in the Fulcher paper)
+dbc = sqlite('D:\McGill\cobralab\PS_Dimensions\gene_mapping\GCEA_data\2022-01-13\GO_2022-01-13.db', 'readonly');
```

Comment:
Don't add absolute hard coded paths, use a generic relative path for someone using this in a project.